services/gateway/README.md

```diff
@@ -83,15 +91,17 @@ OCM_GRPC_PROTOCOL=unix"
 OCM_GRPC_ADDR=/var/run/ocis/ocm.sock"
 GATEWAY_OCM_ENDPOINT=unix:/var/run/ocis/ocm.sock"
 
-// storage system
+// storage related
 STORAGE_SYSTEM_GRPC_PROTOCOL="unix"
 STORAGE_SYSTEM_GRPC_ADDR="/var/run/ocis/storage-system.sock"
-STORAGE_GATEWAY_GRPC_ADDR="unix:/var/run/ocis/storage-system.sock"
-STORAGE_GRPC_ADDR="unix:/var/run/ocis/storage-system.sock"
 SHARING_USER_CS3_PROVIDER_ADDR="unix:/var/run/ocis/storage-system.sock"
 SHARING_USER_JSONCS3_PROVIDER_ADDR="unix:/var/run/ocis/storage-system.sock"
 SHARING_PUBLIC_CS3_PROVIDER_ADDR="unix:/var/run/ocis/storage-system.sock"
 SHARING_PUBLIC_JSONCS3_PROVIDER_ADDR="unix:/var/run/ocis/storage-system.sock"
+
+// the STORAGE_xxx envvars should (and will) be named SETTINGS_STORAGE_xxx
+STORAGE_GATEWAY_GRPC_ADDR="unix:/var/run/ocis/storage-system.sock"
+STORAGE_GRPC_ADDR="unix:/var/run/ocis/storage-system.sock"
```

Comment:
reference the other issue and update this to use the corret env vars. then we can first merge the fix of the env vars and then this PR is already correct.

Reply:
Reference made, see on top.
You need to approve + merge PR: #10362 first.
This is a dev decision, therefore please press the button...
I will do the text fix commit afterwards.

Reply:
done, fixed